test-integration/workflow-samples/angular.json

```diff
@@ -25,7 +25,7 @@
       "app-sample": "ng-mongodb-kafka-cucumber",
       "entity": "mongodb",
       "environment": "dev",
-      "java": "21"
```

Comment:
21 would be still kind of interesting, as it is an LTS supported version, just like jdk 17